### PR TITLE
Migrate poolable elements to new list utilities

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java [hunk 1]
```diff
@@ -128,6 +154,17 @@ public PoolDto updatePool(@PathVariable Long poolId, @RequestBody PoolDto pool)
     return Dtos.asDto(poolService.get(poolId), true);
   }
 
+  @RequestMapping(value = "/{poolId}/contents", method = RequestMethod.POST, produces = "application/json")
```
Comment (djcooke) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java]: Should be `RequestMethod.PUT`

File: miso-web/src/main/webapp/scripts/pool_ajax.js [hunk 2]
```diff
@@ -483,137 +483,6 @@ Pool.ui = {
...
-            "mData": "library"
-          },
-          {
-            "sTitle": "Sample",
-            "sType":"natural",
-            "bSortable": false,
-            "mData": "sample"
-          },
-          {
-            "sTitle": "Indices",
```
Comment (djcooke) [inline miso-web/src/main/webapp/scripts/pool_ajax.js]: This column is missing from the new table. Intentional?

Comment (apmasell) [inline miso-web/src/main/webapp/scripts/pool_ajax.js]: Nope. Oversight. Fixed.

File: miso-web/src/main/webapp/scripts/list_poolelements.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    return [];
+  },
+  createColumns : function(config, projectId) {
+    return [
+        {
+          'sTitle' : 'Dilution Name',
+          'sType' : 'natural',
+          'mData' : 'name',
+          'include' : true,
+          'iSortPriority' : 1
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/list_poolelements.js]: It's sorting weirdly "...,9991, 9990, *999*, 9989,..."

Comment (apmasell) [inline miso-web/src/main/webapp/scripts/list_poolelements.js]: Fixed.

**Pull request merged**